### ISSUE TITLE
Testing on Custom Dataset

### ISSUE THREAD

**Issue by gayatriprasad** [jixunbo/LightMBN#1]
Hello,

Thank you for the repo. I'm thrilled to have multiple datasets and loss functions working together. 

I was just wondering if you have a provision for testing a custom dataset ? And also about how to set-up the custom dataset.

**Comment by jixunbo** [comment]
hi, if you want to set up a custom dataset, please arrange it like dukemtmc-reid dataset:
```
├── DatasetName/
│   ├── bounding_box_test/          /* Files for testing (gallery)
│   ├── bounding_box_train/         /* Files for training 
│   ├── query/                      /* Files for testing (query images)
```
and rename the images as '0001_c2_f0046782.jpg' --'[p_id]_c[cam_id]_[random]'.jpg
then just copy and modify the file 'data_v2--datasets--image--dukemtmcreid.py' as you own dataset preparation file, don't forget to add the name to the '__init__.py'.

besides, since the evaluation performs cross-camera matching, images of the same p_id and cam_id will be discarded, you need to make sure the cam_ids are different for query and gallery.

if you want to train the network on your dataset, I think you probably need to config the total training epochs and batch size.

if you have any further questions, feel free to ask  😄

**Comment by gayatriprasad** [comment]
Hello,
 
Thank you for the quick reply. I will try it this week. Keeping this open for further updates.

**Comment by gayatriprasad** [comment]
Hello, 

I have successfully trained with custom dataset. Thank you. Closing the issue for now.
